### DISCUSSION TITLE
LegacyPlayer logs for Chromie Server.

### DISCUSSION THREAD

**Discussion by HasieEST** [chromiecraft/chromiecraft#4289] [Suggestions]
Pretty simple. Since LegacyPlayer source code is still available, I thought we could have a chromiecraft legacylogs to use, were we could still upload our logs and improve our guilds overall. :)

**Comment by 55Honey** [comment]
Hi Hase,
We're aware of the repository and we'd love to have that working. However resources are crucial and priorities need to be set.

**Reply by HasieEST** [reply]
All cool, just a thought I had, no rush or anything.

**Comment by IskrenPlashkov** [comment]
I was actually going to make the exact same suggestion after AQ40 is clear of bugs.
Just wanna state that Hasie is not the only one who would LOVE to see this implemented.

**Comment by Amrx2** [comment]
Hi Hasie, its now live!  https://legacyplayers.chromiecraft.com/

Will close this one down.

Cheers,
Construct
